wallet-new/src/Cardano/Wallet/API/V1/Migration.hs

```diff
@@ -159,11 +170,11 @@ instance Migrate V0.CAccountId V1.WalletId where
         (walletId, _) :: (V1.WalletId, V1.AccountId) <- eitherMigrate oldAccountId
         pure walletId
 
+instance Migrate V1.PassPhrase Crypto.PassPhrase where
```

Comment:
Are we sure we need this? AFAIK a `V1.PassPhrase` should be exactly equivalent to the Core/Crypto `PassPhrase`. Isn't this the case?